vault/logical_passthrough_test.go

```diff
@@ -86,15 +115,15 @@ func TestPassthroughBackend_Read(t *testing.T) {
 		resp.Secret.InternalData = nil
 		resp.Secret.LeaseID = ""
 		if !reflect.DeepEqual(resp, expected) {
-			t.Fatalf("bad response.\n\nexpected: %#v\n\nGot: %#v", expected, resp)
+			t.Fatalf("bad response.\n\nexpected:\n%#v\n\nGot:\n%#v", expected, resp)
 		}
 	}
 	b := testPassthroughLeasedBackend()
-	test(b, "lease", true)
-	test(b, "ttl", true)
+	test(b, "lease", "1h", true)
+	test(b, "ttl", "5", true)
 	b = testPassthroughBackend()
-	test(b, "lease", false)
-	test(b, "ttl", false)
+	test(b, "lease", int64(10), false)
```

Comment:
No, this ensures that the interface value is the right type. Otherwise this defaults to an int.